n/nbconvert/build_info.json

```diff
@@ -2,13 +2,15 @@
   "maintainer": "stutiibm",
   "package_name": "nbconvert",
   "github_url": "https://github.com/jupyter/nbconvert",
-  "version": "v7.12.0",
+  "version": "v7.16.4",
   "default_branch": "main",
-  "build_script": "nbconvert_ubi_8.7.sh",
   "package_dir": "n/nbconvert/",
+  "build_script": "nbconvert_ubi_9_3.sh",
   "docker_build": false,
-  "v7.7.3, v7.12.0": {
-    "build_script": "nbconvert_ubi_8.7.sh"
+  "validate_build_script": true,
+  "use_non_root_user": false,
+  "v7.16.*": {
+    "build_script": "nbconvert_ubi_9_3.sh"
```

Comment:
"v*.*.*: - this should point to new ubi 9 script

Reply:
Updated build_info.json with required changes.